transformers/integration_tests/vow_file_auction_attributes_test.go

```diff
@@ -71,17 +71,24 @@ var _ = Describe("VowFile LogNoteTransformer", func() {
 		err = tr.Execute(eventLogs)
 		Expect(err).NotTo(HaveOccurred())
 
-		var dbResult []vowFileModel
-		err = db.Select(&dbResult, `SELECT what, data from maker.vow_file`)
+		var msgSenderId int64
+		msgSenderAddress := common.HexToAddress(eventLogs[0].Log.Topics[1].String()).Hex()
```

Comment:
I think we should replace `eventLogs[0].Log.Topics[1].String()` with `0xBE8E3e3618f7474F8cB1d074A26afFef007E98FB` so that we know we're matching the actual address and not failing to parse the topic